Add-ons/UmbracoForms/Developer/Security/index.md

```diff
@@ -51,10 +51,17 @@ With that in place the _Form Security_ tree divides into three sub-trees:
 
 - Under _User Permissions_, each user that has a specific user permission record is listed and can be managed.  Records for users can be created or deleted via the tree's action menu.
 
-- As we no longer necessarily have a user permission record for every user in the system, _Start folders_ are moved into a separate area where all users are listed and hence the appropriate options can be set.
-
 As new forms are created, user groups with aliases listed in the `GrantAccessToNewFormsForUserGroups` configuration setting will be automatically given access.  For example, with a value of `admin,editor`, the built-in Adminstrators and Editors groups would have access.
 
+### Start folders for user groups
+
+In Umbraco Forms 8.12.0 and 9.4.0, start folders were also enabled for user groups.  They work in a similar way to the group based permissions described above:
```

Comment:
In Umbraco Forms 8.12.0 and 9.4.0, start folders are also enabled for User Groups.  They work in a similar way as the group based permissions described above: